ui/src/components/Views/Configs/MountPointForm.tsx

```diff
@@ -41,19 +40,18 @@ export const OnBoarding = (): ReactElement => {
     setUserState({ loading: false, selectedUser: foundUsers[0], users: foundUsers });
   };
 
-  const checkLocalImage = async () => {
-    setHasLocalImage({ checking: true, isPresent: hasLocalImage.isPresent });
-    const images = await ddClient.docker.listImages() as [DockerImage];
-    const isPresent = images.filter(image => image.RepoTags?.at(0).split(':').at(0) === 'localstack/localstack');
-    setHasLocalImage({ checking: false, isPresent: isPresent.length > 0 });
-    return isPresent.length > 0;
-  };
-
   useEffect(() => {
     const execChecks = async () => {
       if (userState.users.length === 0) {
-        const isImagePresent = await checkLocalImage();
-        if (isImagePresent) {
+
+        setHasLocalImage({ checking: true, isPresent: hasLocalImage.isPresent });
+
+        const images = await ddClient.docker.listImages() as [DockerImage];
+        const isPresent = images.some(image => image.RepoTags?.at(0) === LATEST_IMAGE);
```

Comment:
should we also check if the pro image is present? 
since we introduced updating for both images..

Reply:
Not everybody who uses the extension might be a PRO user so if I added that they might download something that will never be used... Also the main reason for this check is that I use `localstack/localstack` to both spawn the CLI and to check the user home dir , that's why it's necessary to have it:+1: